chapters/5-snippet-information.md

```diff
@@ -434,4 +434,36 @@ Example:
 
     <Snippet rdf:about="...">
         <name>from linux kernel</name>
+    </Snippet>
+
+## 5.11 Snippet Attribution Text <a name="5.11"></a>
+
+**5.11.1** Purpose: This field provides a place for the SPDX data creator to record, at the snippet level, acknowledgements that may be required to be communicated in some contexts. This is not meant to include the snippet's actual complete license text (see `LicenseConcluded` and `LicenseInfoInSnippet`), and may or may not include copyright notices (see also `SnippetCopyrightText`). The SPDX data creator may use this field to record other acknowledgements, such as particular clauses from license texts, which may be necessary or desirable to reproduce.
```

Comment:
@goneall I believe `LicenseConcluded` is the File version of this tag -- should it be `SnippetLicenseConcluded` here instead?

Reply:
@swinslow Good catch!  I just pushed a fix.

Reply:
looks good to me, thanks!